src/ucm/mmap/install.c

```diff
@@ -273,36 +282,53 @@ static ucs_status_t ucs_mmap_install_reloc(int events)
     return UCS_OK;
 }
 
+static int ucm_mmap_events_to_native_events(int events)
+{
+    int native_events = events;
+
+    if (events & UCM_EVENT_VM_MAPPED) {
```

Comment:
what about removing aggregate events, and removing memtype events?

Reply:
we don't need this: function for testing hooks and for installing hooks are operate by own set of flags (including or excluding aggregate events). Adding additional processing will just complicate logic

Reply:
what about memtype? they would be added into installed events, even though they are not tested/installed

Reply:
yep, fixed